packages/react/src/components/FEaaS/BYOCWrapper.test.tsx

```diff
@@ -257,19 +257,20 @@ describe('Error handling', () => {
   // Disabling these two tests until we have AppRouter and can switch back from clientFallback to fallback
   // Components does some workarounds to support client BYOC and client fallback without error frame flickering during component load
   // This results in fallback frame being rendered post-hydration, and not being findable in test context.
-  xit('should render missing component frame when component is not registered', async () => {
+  it('should render missing component frame when component is not registered', async () => {
     const props = {
       params: { ComponentName: 'NonExistentComponent' },
       components: {},
       fetchedData: {},
     };
 
     const wrapper = render(<BYOCComponent {...props} />);
-
-    expect(wrapper.queryAllByText('This component was not registered').length).to.equal(1);
+    waitFor(() => {
+      expect(wrapper.queryAllByText('This component was not registered').length).to.equal(1);
```

Comment:
this assertion isn’t being executed - I can change its value to anything and the test still passes.
Most likely yoiu forgot to add `await` before `waitFor`